services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -659,7 +659,15 @@ module.exports = {
         maxResults,
         ...(pageToken && { pageToken }),
       };
-      return googleDataApiClient.request('youtube.playlistItems', payload);
+      try {
+        const response = await googleDataApiClient.request('youtube.playlistItems', payload);
+        return {
```

Comment:
Spread the entire response and then override the `items` property. This way, the entire response object is preserved as-is. Otherwise other properties of the response object will be lost.

```js
return {
  ...(response || {}),
  items: getAsArray(response, 'items').filter(video => (input.showPrivate ? video : get(video, 'status.privacyStatus') === 'public')),
};
```

Reply:
Also, the reason to use `...(response || {})` is because the spread operator allows for `undefined` and any object... however, a `null` value (or any other falsey value) would error. This converts the falsey value to an empty object to prevent that.